alacritty_terminal/src/term/cell.rs

```diff
@@ -8,6 +8,8 @@ use crate::ansi::{Color, NamedColor};
 use crate::grid::{self, GridCell};
 use crate::index::Column;
 
+use vte::ansi::Hyperlink as VteHyperlink;
```

Comment:
This import needs to go up to the other foreign crate imports.

Reply:
Fixed in https://github.com/alacritty/alacritty/pull/6896/commits/fb7a5d4d9c92fd4389bc59020a629e108a87aa35.